packages/design-system/src/Modal/Modal.styles.tsx

```diff
@@ -0,0 +1,80 @@
+import styled, { keyframes } from "styled-components";
+import { Overlay, Content, Close } from "@radix-ui/react-dialog";
+import {
+  ModalContentBodyClassName,
+  ModalContentFooterClassName,
+} from "./Modal.constants";
+
+const overlayShow = keyframes`
+  from {
+    opacity: 0;
+  }
+  to {
+    opacity: 1;
+  }
+`;
+
+const contentShow = keyframes`
+  from {
+    opacity: 0;
+    transform: translate(-50%, -50%) scale(0.9);
+  }
+  to {
+    opacity: 1;
+    transform: translate(-50%, -50%) scale(1);
+  }
+`;
+
+export const StyledOverlay = styled(Overlay)`
+  /* TODO: Opacity for colors needs to be handled in a better way*/
+  background-color: #39393999;
+  position: fixed;
+  inset: 0;
+  animation: ${overlayShow} 250ms cubic-bezier(0.16, 1, 0.3, 1);
+`;
+
+export const StyledContent = styled(Content)`
+  background-color: var(--ads-v2-color-bg);
+  border-radius: var(--ads-v2-border-radius);
+  position: fixed;
+  top: 40%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  width: 50vw;
+  max-width: 90vw;
+  max-height: 85vh;
+  padding: var(--ads-v2-spaces-6) var(--ads-v2-spaces-5);
+  font-family: var(--ads-v2-font-family);
```

Comment:
We don't want font family and color no? We don't want to influence the styles of the children of Panel, just the panel itself.